bin/combinejsons.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python
+import json
+import argparse
+
+# Function to load JSON data from a file
+def load_json_file(file_path):
+    with open(file_path, 'r') as file:
+        return json.load(file)
+
+# Function to save JSON data to a file
+def save_json_file(data, file_path):
+    with open(file_path, 'w') as file:
+        json.dump(data, file, indent=4)
+
+# Main function to combine the Info JSON and the genotype JSON
+def combine_json_files(info_json_file, genotype_json_file, output_file):
+    # Load the Info and genotype JSON files
+    info_data = load_json_file(info_json_file)
+    genotype_data = load_json_file(genotype_json_file)
+
+    # Combine the data by appending the genotype JSON
+    combined_data = {
+        **info_data,
+        "isnp_genotype": genotype_data
```

Comment:
```suggestion
        "id_snp_genotypes": genotype_data
```

Sorry, I want the var names in CDM to be super-clear and self-explanatory. Might seem a nitpick, but makes a difference later down the line, both when working with the QC data in the CDM client and in the future when I get questions about specific QC vars.

Reply:
Thank you for the suggestion. I will make sure it is same and clearly explanatory :)